app/assets/javascripts/services/search.js

```diff
@@ -0,0 +1,10 @@
+angular.module("OpenHq").factory("Search", function(railsResourceFactory) {
+
+  return railsResourceFactory({
```

Comment:
As we’ll probably only ever have 1 endpoint for search we should just use `$http` directly, something like:

```js
$http({
  method: "GET",
  url: "/api/v1/search",
  params: {
    term: term,
    page: page
  }
}).then(function(resp) {
  return resp.search_documents;
});
```